velox/exec/RowContainer.h

```diff
@@ -367,10 +363,11 @@ class RowContainer {
         auto row = iter->rowOffset;
         while (row + rowSize <= limit) {
           rows[count++] = data + row +
-              (iter->normalizedKeysLeft > 0 ? sizeof(normalized_key_t) : 0);
+              (iter->normalizedKeysLeft > 0 ? normalizedKeySize() : 0);
           row += rowSize;
+          auto totalBytes2 = totalBytes + rowSize;
```

Comment:
> totalBytes2

Would it be possible to come up with a more descriptive name? Should we add a test case for the changes in RowContainer?